contracts/ColonyFunding.sol

```diff
@@ -279,12 +295,8 @@ contract ColonyFunding is ColonyStorage, DSMath {
   taskExists(_id)
   taskNotFinalized(_id)
   {
-    Task storage task = tasks[_id];
-    uint currentAmount = task.payouts[_role][_token];
-    task.payouts[_role][_token] = _amount;
-
-    uint currentTotalAmount = task.totalPayouts[_token];
-    task.totalPayouts[_token] = add(sub(currentTotalAmount, currentAmount), _amount);
+    uint currentTotalAmount = getTotalTaskPayout(_id, _token);
```

Comment:
You're saying make a note to not delete this call, even if we don't end up using the variable down the road, because the function call itself performs an important check.

Reply:
Yup